libs/sdk-bindings/bindings-swift/Sources/BreezSDKNotification/ServiceLogger.swift

```diff
@@ -0,0 +1,45 @@
+import Foundation
+import os.log
+
+#if DEBUG && true
+fileprivate var os_log = Logger(
+    subsystem: Bundle.main.bundleIdentifier!,
+    category: "ServiceLogger"
+)
+#else
+fileprivate var os_log = Logger(OSLog.disabled)
+#endif
+
+class ServiceLogger {
```

Comment:
Same here, I don't see why we need the ServiceLogger class instead of directly use LogStream.

Reply:
Resolving as discussed on Slack. 

Decided to keep and make changes that makes ServiceLogger an internal class on iOS, the behavior is consistent with Android this way.

- Re-added `setLogger` on iOS that takes a `LogStream` to initialize `ServiceLogger`.
- Removed tag from `LogEntry` to avoid unwanted changes for SDK users
  - Default logs will have proper tags but the user has to set an umbrella tag if they use their custom LogStream.